README.md

```diff
@@ -37,6 +37,44 @@ You can configure default value of analyze settings like example below.
 ```jsonc
 "WavPreview.analyzeDefault": {
         // Settings about WaveForm
+        /*
+         Make the waveform visible or hidden.
+         true: visible, false: hidden
+         default: true
+        */
+        "waveformVisible": true,
+
+        /*
+         Adjust height of the waveform. 
+         The valid range of [0.3, 2.0] scales the default height.
+         default: 1.0
+         This option can only be configured through the settings file.
+        */
+        "waveformVerticalScale": 1.0,
+
+        /*
+         Make the waveform channel label shown or hidden.
+         true: shown, false: hidden
+         default: false
+        */
+        "waveformShowChannelLabel": false,
```

Comment:
Channel labels are also convenient, so I think it's okay to always display them. I don't need to be able to change it in the settings, so could you make it always visible? The same goes for spectrogram.